docs_src/design/adr/device-service/0011-DeviceService-Rest-API.md

```diff
@@ -79,12 +79,12 @@ The values obtained when readings are taken, or used to make settings, are expre
 
 | Type | EdgeX types | Representation
 | --- | --- | ---
-| Boolean | `bool` | "true" or "false"
-| Integer | `uint8-uint64`, `int8-int64` | Numeric string, eg "-132"
-| Float | `float32`, `float64` | base64 encoded little-endian binary or decimal with exponent, eg "1.234e-5"
-| String | `string` | string
-| Binary | `bytes` | octet array
-| Array | `boolarray`, `uint8array-uint64array`, `int8array-int64array`, `float32array`, `float64array` | JSON Array, eg "["1", "34", "-5"]"
+| Boolean | `Bool` | "true" or "false"
+| Integer | `Uint8-Uint64`, `Int8-Int64` | Numeric string, eg "-132"
+| Float | `Float32`, `Float64` | base64 encoded little-endian binary or decimal with exponent, eg "1.234e-5"
```

Comment:
Based on the F2F, shouldn't we remove any references to base64 float encoding? We probably need to scrub the Core-Data V2 OpenAPI documentation as well.